akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -110,8 +113,7 @@ class Http2ServerSpec extends AkkaSpec("" + "akka.loglevel = debug")
           expectedRequest = HttpRequest(
             method = HttpMethods.GET,
             uri = "http://www.example.com/",
-            // FIXME: should be modeled header: headers.`Cache-Control`(CacheDirectives.`no-cache`()) :: Nil,
-            headers = RawHeader("cache-control", "no-cache") :: Nil,
+            headers = Vector(`Cache-Control`(`no-cache`)),
```

Comment:
What do you mean exactly? That people might be tripped up about getting a lower-cased `RawHeader`?